src/app/admin/components/reports/reports.directive.js

```diff
@@ -398,6 +405,8 @@
                         var statusActivity = angular.copy(activity);
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
+
+                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
```

Comment:
Why are you marking this activity as questionable? That's what the class "bg-danger" implies...